src/lang/es.json

```diff
@@ -1545,7 +1545,7 @@
   "Next week": "La semana que viene",
   "Next {n} weeks": "Próximas {n} semanas",
   "Nice": "Bonito",
-  "nice_ride": "¡Bien {0}!",
+  "nice_ride": "¡Buen/a {0}!",
```

Comment:
Got this email notification today:

![image](https://github.com/user-attachments/assets/0dbb0a75-7ef5-4308-a1bd-95a36960dda7)

"Bien" is an adverb and can't describe nouns like "ciclismo"; we need the adjective "buen" to say "buen ciclismo".